.github/workflows/fossa.yml

```diff
@@ -0,0 +1,20 @@
+name: fossa
+
+on:
+  push: {}
+  pull_request: {}
+
+jobs:
+  fossa-scan:
+    runs-on: ["org--${{ github.repository_owner }}--amd64--containers"]
+    container: ubuntu-22.04
```

Comment:
I'm not familiar with this image, I've only used ubuntu-latest so far. Can you show me where that it used?

Reply:
I'm now using a container recommended by Chris Olstrom because it contains the dev tools and certs I need.

Reply:
Resolved by just running on `ubuntu-latest`